backend/.eslintrc.js

```diff
@@ -18,5 +18,6 @@ module.exports = {
   rules: {
     'prettier/prettier': 'error',
     'no-console': 'off',
+    'import/prefer-default-export': 'off',
```

Comment:
Should this be disabled for the entire project and not individual files?

Reply:
Yea I guess you're right. I'll fix it.